tests/utilities/test_ast_from_value.py

```diff
@@ -86,6 +86,14 @@ def converts_float_values_to_float_asts():
 
         assert ast_from_value(1e40, GraphQLFloat) == FloatValueNode(value="1e+40")
 
+        assert ast_from_value(123456789.01234567, GraphQLFloat) == FloatValueNode(
+            value="123456789.01234567"
+        )
+
+        assert ast_from_value(1.0, GraphQLFloat) == FloatValueNode(value="1")
```

Comment:
We already have a similar test above, but it will not harm adding this one as well.